src/components/Greetings.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from './store/greetingsSlice';
+
+function Greetings() {
+  const dispatch = useDispatch();
+  const greetingsCont = useSelector((state) => state.greetings.greetings);
+
+  useEffect(() => {
+    dispatch(fetchGreetings());
+  }, [dispatch]);
+
+  return (
+    <div>
+      {greetingsCont.length > 0 && (
+        <h2>
+          {
+            greetingsCont[Math.floor(Math.random() * greetingsCont.length)]
+              .greeting
```

Comment:
- [ ] Please consider the change on the backend, you should receive the random greeting from the response of the server, instead of calculating it from the client! 🤓 🔧